### ISSUE TITLE
mima failing in 2.11.x-jdk8 build

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#556]
```
[info] >>> The dbuild result is------------: FAILED (failed: mima)
```

**Comment by SethTisue** [comment]
as of typesafehub/migration-manager@f29a309582357fe83e81ea8d8ffbb878c7099228 MiMa cross builds on 2.10 and 2.12, but not 2.11

so it would reasonable to drop it from the 2.11 build entirely, but I think we might as well leave it in; it's sufficient to just freeze it at the last green commit

**Comment by SethTisue** [comment]
fixed by 87bc75f6ef517f4d348a9701bf812ef0258436bf
